invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -25,8 +26,6 @@
 from invenio_records_resources.services.errors import PermissionDeniedError
 from sqlalchemy.orm.exc import NoResultFound
 
-from invenio_app_rdm.urls import record_url_for
```

Comment:
This is 1 of the 2 main integrations for this module: replacing the `urls.py` module completely.